src/porepy/numerics/fv/fv_elliptic.py

```diff
@@ -344,3 +347,429 @@ def enforce_neumann_int_bound(
         """
         # Operation is void for finite volume methods
         pass
+
+
+class FVVectorElliptic(pp.numerics.mixed_dim.solver.Solver):
```

Comment:
Do you actually use this class for anything? If not, it feels like a premature abstraction that may cause more harm than good in the future. As an example, it indicates that mappings between grids in different dimensions is straightforward for elasticity - fact is that nobody knows this.

Unless I am overlooking something here, my preference would be to include what is needed to get contact mechanics working into mpsa, and then remove the rest - but save the commit id.

Reply:
I kind off agree, but I don't see the harm in the abstraction. If you want to couple different dimensions you first need to write a mixed dim interface law, and if you do that you know enough to know what you are doing.  But I agree most (or all) can be moved to the Mpsa-class. However, I think we should add a VectorEllipticDiscretization to the mixed-dim folder (this folder should not be called mixed-dim, but that is an other issue) as @IvarStefansson  as plans of finite elements for elasticity.

Reply:
Also, this specify the contract between the node-discretization and interface law. Keeping this separate from mpsa should be beneficial.